docs/html/development/index.rst

```diff
@@ -28,3 +29,4 @@ or the `distutils-sig mailing list`_, to ask questions or get involved.
 
 .. _`on Libera.chat`: https://kiwiirc.com/nextclient/#ircs://irc.libera.chat:+6697/pypa-dev
 .. _`distutils-sig mailing list`: https://mail.python.org/mailman3/lists/distutils-sig.python.org/
+.. _`Discord`: https://discord.com/channels/803025117553754132/837243676814999553
```

Comment:
:thinking: actually, how accessible is this link if I'm not already a user there. Maybe we should just link to the `pypa` discord link: https://discord.gg/pypa since that leads to an invite, and point people to the channel